common/spring-cloud-stream-modules-common-configuration/pom.xml

```diff
@@ -29,6 +29,12 @@
 			<optional>true</optional>
 		</dependency>
 		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-sftp</artifactId>
```

Comment:
I followed the FTP model - they are in common because the connection factory configuration is common across source and sink.

They are marked as optional so they won't be pulled in when not needed.